main.py

```diff
@@ -1,10 +1,43 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        for skill in player_data["race"]["skills"]:
+            Skill.skill_import(
+                skill_name=skill["name"],
+                skill_bonus=skill["bonus"],
+                skill_race=Race.race_import(
```

Comment:
You end up querying the database for the same `Race` twice. Why don't you store it in a variable?